### PR TITLE
Raven Slingshot Credentials

### PR DESCRIPTION
**Pull request by Vinniekun**
Added Raven Slingshot Credentials API requests to the Provider Credentials

### CODE REVIEW

**Comment by Vinniekun** [comment]
[[Usermanagement] Add RavenSlingshot Provider credentials](https://trello.com/c/jboxWXSG/218-usermanagement-add-ravenslingshot-provider-credentials)

**Comment by Vinniekun** [comment]
[[Usermanagement] Add RavenSlingshot Provider credentials](https://trello.com/c/fG8Ltx0i/215-usermanagement-add-ravenslingshot-provider-credentials)

File: Leaf API.postman_collection.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"info": {
-		"_postman_id": "ad99cd82-f090-4bcb-aa9b-a1fef67d12ba",
+		"_postman_id": "1600ce4b-1270-4cf4-8a63-9ae15aba3e0b",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:3]: we don't have to commit this change

Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:3]: We don't have to commit this change!

File: Leaf API.postman_environment.json [hunk 2]
```diff
@@ -231,9 +231,24 @@
 			"key": "raven_refresh_token",
 			"value": "Raven refresh token obtained from step 2 \"Get Raven Token\" (auto)",
 			"enabled": true
+		},
+		{
+			"key": "ravenslingshot_api_key",
+			"value": "fill with API Key obtained from Raven Slingshot",
```
Comment (fabyan-alexander) [inline Leaf API.postman_environment.json:237]: Perfect description!

File: Leaf API.postman_environment.json [hunk 3]
```diff
@@ -231,9 +231,24 @@
...
+			"enabled": true
+		},
+		{
+			"key": "ravenslingshot_access_key",
+			"value": "Access key received from Raven Slingshot portal",
+			"enabled": true
+		},
+		{
+			"key": "ravenslingshot_shared_secret",
+			"value": "fill with your shared secret obtained from Raven Slingshot's API owner",
```
Comment (fabyan-alexander) [inline Leaf API.postman_environment.json:247]: Perfect description!

File: Leaf API.postman_environment.json [hunk 4]
```diff
@@ -231,9 +231,24 @@
...
 			"enabled": true
+		},
+		{
+			"key": "ravenslingshot_api_key",
+			"value": "fill with API Key obtained from Raven Slingshot",
+			"enabled": true
+		},
+		{
+			"key": "ravenslingshot_access_key",
+			"value": "Access key received from Raven Slingshot portal",
```
Comment (fabyan-alexander) [inline Leaf API.postman_environment.json:242]: We could mention that this information is obtained by the user/farmer account

File: Leaf API.postman_collection.json (hunk 1 above)
Comment (Vinniekun) [reply Leaf API.postman_collection.json]: Roger that! https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/52/commits/503338327af57c67b0d602dc17becfca2e1b37cb

**Review by fabyan-alexander — APPROVED** [review]

**Review by palomacione — APPROVED** [review]

**Pull request merged**